iib/workers/tasks/fbc_utils.py

```diff
@@ -91,7 +91,7 @@ def merge_catalogs_dirs(src_config: str, dest_config: str):
     shutil.copytree(src_config, dest_config, dirs_exist_ok=True)
 
 
-def extract_fbc_fragment(temp_dir: str, fbc_fragment: str) -> Tuple[str, str]:
+def extract_fbc_fragment(temp_dir: str, fbc_fragment: str) -> Tuple[str, List[str]]:
     """
     Extract operator package from the fbc_fragment image.
 
```

Comment:
return: fbc_fragment path, fbc_operator_package
```suggestion
return: fbc_fragment path, fbc_operator_packages
:rtype: list
```